website/docs/integrator-section/verifiable-document/did/revoking-document.md

```diff
@@ -59,19 +59,12 @@ It is possible to revoke a signed document only if you fulfill certain **prerequ
 open-attestation document-store revoke --address <DOCUMENT_STORE_LOCATION> --hash <HASH_OF_DOC(S)>  --network <NETWORK> --encrypted-wallet-path <PATH_OF_WALLET>
 ```
 
-### Frequently asked questions (FAQ)
-
-Q: [This article](/docs/docs-section/how-does-it-work/comparison#price) mentioned that if I use DID documents, I will not need to pay for transactions. But following this procedure, why do I still have to pay for at least 1 transaction (deploying a `documentStore`)?
-
-A: Yes, that's correct. In the current version, this implementation will still need at least 1 transaction to the Ethereum blockchain.
-
-Q: Although I did not issue any documents from the deployed `documentStore`, how can revoke this document from the same `documentStore`?
-
-A: That's because the revocation mapping in the `documentStore` is separate from the issued mapping.
-
+>**Note:** 
```

Comment:
There is another line above this `**Note:** Before running the command, replace the variables with the actual names to fit your case.`

<img width="862" alt="Screenshot 2023-10-27 at 3 19 25 PM" src="https://github.com/Open-Attestation/documentation-website/assets/31716292/6fb88256-4e2e-4e74-a491-941b91b9c376">


Looks weird having 2 notes so close to each other, also I think the note above can be removed. In other sections I see that we either explain what each parameter is e.g. Under DID -> Signing Documents we say `In the example above: ...` 

And under DID -> Wrapping Documents we say `Replace the folder/file names below and run the command:` 

Can we follow one of these 2 and remove this note? We should stick to 1 or at most 2 phrases for these instructions. My personal opinion is that the way we do it in DID -> Signing Documents is the best.

Reply:
Removed the first note and updated the description for consistency